CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Contributing to Chunk
+
+Write and run tests, and use the auto-format as well as the linter tools:
```

Comment:
Maybe point out how to install staticcheck and Go?

Just created https://github.com/cuducos/chunk/issues/16 to make sure the pre-commit still under our radar.